config/application.rb

```diff
@@ -47,6 +47,8 @@ class Application < Rails::Application
         resource '/master_files/*/structure.json', headers: :any, methods: [:get, :post, :delete]
         resource '/master_files/*/waveform.json', headers: :any, methods: [:get]
         resource '/master_files/*/*.m3u8', headers: :any, credentials: true, methods: [:get, :head]
+        resource '/master_files/*/caption_manifest', headers: :any, methods: [:get]
+        resource '/master_files/*/captions', headers: :any, methods: [:get]
```

Comment:
Not sure if necessary for final implementation. These are necessary when the HLS manifest is requested by an external player such as Ramp, but depending on how iOS handles the requests it may not  be through CORS. If we are able to provide captions to IIIF players via the IIIF manifest, then the captions being included in the HLS manifest may only be necessary for iOS.